c7n/cli.py

```diff
@@ -98,6 +99,10 @@ def _default_options(p, exclude=[]):
             help="Cache validity in minutes (default %(default)i)")
     else:
         p.add_argument("--cache", default=None, help=argparse.SUPPRESS)
+    if 'session-policy' not in exclude:
+        p.add_argument("--session-policy", required=False, default=None,
+                       action=LoadSessionPolicyJson,
+                       help="[OPTIONAL] Policy Document to be used as a session policy")
```

Comment:
this probably needs a note in the help or prefix on the parameter name re `aws` we are able to use reuse the other cli parameters (assume, region, etc) across providers atm, but this one is fairly aws specific.

Reply:
Good point, addressed.